Ladybird/Settings.cpp

```diff
@@ -23,4 +23,14 @@ void Settings::set_homepage(QString const& homepage)
     m_qsettings->setValue("homepage", homepage);
 }
 
+QString Settings::page_new_tab()
```

Comment:
`new_tab_page` sounds more natural to me; yet iʼm not a native language user and can mislead you here… 😞

Reply:
I would also call it the "new tab page".